seqr/models.py

```diff
@@ -64,6 +64,10 @@ def _compute_guid(self):
     def __unicode__(self):
         return self.guid
 
+    def __str__(self):
```

Comment:
You took this out of the matchmaker models be left it here - how come?

Reply:
The matchmaker models inherit `seqr.models.ModelWithGUID`. Adding `__str__` to the base class to avoid adding it all sub-classes.

Reply:
Oh i misread this, Ithought it was in the Individual model still